module/documents/actor.mjs

```diff
@@ -147,12 +154,28 @@ export default class ActorEd extends Actor {
       target: { base: difficulty },
       karma: { pointsUsed: this.system.karma.useAlways ? 1 : 0, available: this.system.karma.value, step: this.system.karma.step },
       devotion: { available: this.system.devotion.value, step: this.system.devotion.step },
-      chatFlavor: equipment.name + " Equipment Test",
+      chatFlavor: sourceActor + " " + game.i18n.localize( "ED.Rolls.rolls" ) + " " + equipment.name + " Test" + targetActors,
     } );
     const roll = await RollPrompt.waitPrompt( edRollOptions, options );
     this.#processRoll( roll );
   }
 
+    /**
+   * @summary                       get the Target of the action.
+   * @param {string} name           The name of the target
+   * @returns {string}              Returns a string of target actors
+   */
+    targetNames( ) {
```

Comment:
I don't think we need an extra function for that. Seems like something that is only used in that one particular place. Rather pass the targets to a formatted i18n string (`game.i18n.format("LocalizationKey", targets)`)

Reply:
it is used in Roll Attribute, Roll Equipment and Roll ability, läidder auch in ÄTTACK un so

Reply:
added additional commit